--- openvas-omp.rb.orig	2015-07-28 23:25:17.147357049 -0700
+++ openvas-omp.rb	2015-07-28 23:03:48.193453494 -0700
@@ -571,9 +571,9 @@
         resp.elements.each('//get_reports_response/report') do |report|
           td = Hash.new
           td["id"] = report.attributes["id"]
-          td["task"] = report.elements["report/task/name"].text
-          td["start_time"] = report.elements["report/scan_start"].text
-          td["stop_time"] = report.elements["report/scan_end"].text
+          td["task"] = report.elements["task/name"].to_s
+          td["start_time"] = report.elements["scan_start"].to_s
+          td["stop_time"] = report.elements["scan_end"].to_s
           list.push td	
         end
         @reports = list
